src/SeleniumLibrary/keywords/window.py

```diff
@@ -0,0 +1,221 @@
+# Copyright 2008-2011 Nokia Networks
+# Copyright 2011-2016 Ryan Tomac, Ed Manlove and contributors
+# Copyright 2016-     Robot Framework Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from selenium.common.exceptions import NoSuchWindowException
+
+from SeleniumLibrary.base import keyword, LibraryComponent
+from SeleniumLibrary.locators import WindowManager
+
+
+class WindowKeywords(LibraryComponent):
+
+    def __init__(self, ctx):
+        LibraryComponent.__init__(self, ctx)
+        self._window_manager = WindowManager(ctx)
+
+    @keyword
+    def select_window(self, locator='MAIN'):
+        """Selects browser window matching ``locator``.
+
+        If the window is found, all subsequent commands use the selected
+        window, until this keyword is used again. If the window is not
+        found, this keyword fails. The previous window handle is returned,
+        and can be used to return back to it later.
+
+        Notice that in this context _window_ means a pop-up window opened
+        when doing something on an existing window. It is not possible to
+        select windows opened with `Open Browser`, `Switch Browser` must
+        be used instead. Notice also that alerts should be handled with
+        `Handle Alert` or other alert related keywords.
+
+        The ``locator`` can be specified using different strategies somewhat
+        similarly as when `locating elements` on pages.
+
+        - By default the ``locator`` is matched against window handle, name,
+          title, and URL. Matching is done in that order and the the first
```

Comment:
Comma after word title is not needed.

Reply:
Not needed but recommended: http://www.getitwriteonline.com/archive/052709serialcomma.htm